src/main/java/com/isa/morswiny/users/User.java

```diff
@@ -20,18 +20,21 @@
     private String login;
     private String email;
     private String password;
-    private LocalDateTime birthday;
+    private UserTypes userTypes;
+    private LocalDate birthday;
     private List<Event> favourites = new ArrayList<>();
     private List<Event> myEvents = new ArrayList<>();
```

Comment:
nie wiem czy nie lepiej od razu dzialac na Setach by unikac duplikacji eventow i poakzywac, ze umiemy pracowac na czyms wiecej niz array listy :-) mozemy oczywiscie przy listach zostac, ale to wymaga np. "sztucznego" tworzenia logiki unikania duplikatow